### DISCUSSION TITLE
chenge emulator

### DISCUSSION THREAD

**Discussion by aminbouhamed** [GoogleChrome/lighthouse#15248] [General] [answered]
helloo the result  is Emulated Moto G Power with Lighthouse 10.3.0 can i emulate with desktop and how can i do that please , knowing that am using git programatically as node module

**Comment by adamraine** [comment, marked as answer]
You can use the desktop config to generate a report for a desktop site:

```js
import lighthouse, {desktopConfig} from 'lighthouse';

await lighthouse('https://example.com', undefined, desktopConfig);
```

**Reply by aminbouhamed** [reply]
Thank you !

**Comment by raselqabd** [comment]
import lighthouse, {desktopConfig} from 'lighthouse';

await lighthouse('https://example.com', undefined, desktopConfig);
